docs/ext/commands/help.rst

```diff
@@ -0,0 +1,78 @@
+.. currentmodule:: discord
+
+.. _ext_commands_help:
+
+Help Commands
+=================
+
+The simplicity of :class:`~.commands.HelpCommand` allows you to customise
+your experience very easily.
+
+Sending Help
+---------------
+
+You may want to send help if the user doesn't use your command
+correctly. Here is an example of what you can do:
+
+.. code-block:: python3
+    :caption: sos.py
+    :emphasize-lines: 8, 10
+
+    from discord.ext import commands 
+
+    @commands.command()
```

Comment:
```suggestion
    @bot.command()
```

This is consistent with the rest of the commands documentation with bare command examples.